eng/pipelines/common/device-tests-steps.yml

```diff
@@ -86,6 +86,17 @@ steps:
     condition: and(succeeded(), eq(variables['Platform.Name'], 'Mac'))
     retryCountOnTaskFailure: 1
 
+
+  - ${{ if eq(parameters.platform, 'ios')}}:
+    - bash: |
+        rm -r $HOME/Library/Logs/CoreSimulator/*
+        rm -r $HOME/Library/Logs/DiagnosticReports/*
+        zip -9r "$(LogDirectory)/CoreSimulatorLog.zip" "$HOME/Library/Logs/CoreSimulator/"
+        zip -9r "$(LogDirectory)/DiagnosticReports.zip" "$HOME/Library/Logs/DiagnosticReports/"
```

Comment:
It's early in the morning for me here, but aren't you first deleting the directories, and then zipping them up? How is that going to gather anything at all?

Reply:
aw crap
I really didn't think through this at all did I :-/